card/spec/lib/card/reference_spec.rb

```diff
@@ -16,11 +16,11 @@
     end
 
     it "on template creation" do
-      Card.create! :name=>"SpecialForm", :type=>'Cardtype'
-      Card.create! :name=>"Form1", :type=>'SpecialForm', :content=>"foo"
+      Card.create! name: "SpecialForm", type: 'Cardtype'
+      Card.create! name: "Form1", type: 'SpecialForm', content: "foo"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.